orders/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+class Shopping_Cart(models.Model):
+    user         = models.ForeignKey('users.User', on_delete=models.CASCADE)
+    product      = models.ForeignKey('products.Product', on_delete=models.CASCADE)
+    payment_type = models.ForeignKey('Payment_Type',on_delete=models.CASCADE)
+    order_status = models.ForeignKey('Order_Status',on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'shopping_carts'
+
+class Payment_Type(models.Model):
```

Comment:
클래스명에는 `_` 없이 만들어주세요!
```suggestion
class PaymentType(models.Model):
```